doc/man/8/rados.rst

```diff
@@ -26,6 +26,10 @@ Options
 
    Interact with the given pool. Required by most commands.
 
+.. option:: --pgid
+
+   Interact with the given PG id. Required by rados ls command.
```

Comment:
it's not "required", also it applies to all commands which uses an io_ctx in the rados cli tool. we probably could put this way

> As an alternative to `--pool`, `--pgid` also allows user to specify the PG id to specify the pool to which the command will be directed. With this option, certain commands like `ls` allows user to limit the scope of the command. See

and we need to revise the help of `ls` command

> :command:`ls` *outfile*
>     List objects in given pool and write to outfile. if `--pgid` is specified, `ls` will only list the objects in the given PG.